soc/nordic/nrf54h/soc.c

```diff
@@ -52,13 +60,20 @@ static void power_domain_init(void)
 	 *  WFI the power domain will be correctly retained.
 	 */
 
-	nrf_lrcconf_poweron_force_set(NRF_LRCCONF010, NRF_LRCCONF_POWER_DOMAIN_0,
-				      !IS_ENABLED(CONFIG_SOC_NRF54H20_CPURAD));
-	nrf_lrcconf_poweron_force_set(NRF_LRCCONF010, NRF_LRCCONF_POWER_MAIN,
-				      !IS_ENABLED(CONFIG_SOC_NRF54H20_CPURAD));
-
+#if !IS_ENABLED(CONFIG_SOC_NRF54H20_CPURAD)
+	sys_snode_t *node;
+#if IS_ENABLED(CONFIG_PM) || IS_ENABLED(CONFIG_POWEROFF)
+	node = pm_sys_node_id_get();
+#else
+	node = &soc_node;
+#endif
```

Comment:
This looks like a leftover from some previous approach. `node` is not used by the code below.